.gitignore

```diff
@@ -0,0 +1,4 @@
+.DS_Store
+.*.sw?
+.env
+node_modules/
```

Comment:
Nice, this should be fine as the `Dockerfile` does an `nom install`. Should we also delete them from the repo (e.g. look at dat PR diff :D)

Reply:
Also, it's considered a [best practice](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#general-guidelines-and-recommendations) to have a `.Dockerignore` file as it reduces the build time by not transferring a lot of stuff in the context when it's actually not used.

Reply:
no reason to have all those files in the repo

Reply:
That's a good point